### PR TITLE
add methods without style

### PR DESCRIPTION
**Pull request by Anton-Zhytonbaiev**
[DEMO LINK](https://Anton-Zhytonbaiev.github.io/react_list-of-goods/)

### CODE REVIEW

File: src/App.tsx [hunk 1]
```diff
@@ -26,70 +27,124 @@ type ReorderOptions = {
...
-  // eslint-disable-next-line no-console
-  console.log(sortType, isReversed);
+  visibleGoods.sort((a, b) => {
+    switch (sortType) {
+      case SortType.ALPHABET:
+        return a.localeCompare(b);
+
+      case SortType.LENGTH:
+        return a.length - b.length;
+
```
Comment (vanst1) [inline src/App.tsx:43]:
```suggestion
case SortType.NONE:
        return 0;
```

File: src/App.tsx [hunk 2]
```diff
@@ -26,70 +27,124 @@ type ReorderOptions = {
...
+      default:
+        return 0;
```
Comment (vanst1) [inline src/App.tsx:45]: better throw error here

**Review by vanst1 — CHANGES_REQUESTED** [review]
Good job)
Some fix needed:)

File: src/App.tsx [hunk 3]
```diff
@@ -26,70 +27,129 @@ type ReorderOptions = {
...
+            type="button"
+            className={classNames(
+              'button is-warning',
+              { 'is-light': !isReversed },
+            )}
+            onClick={this.reverse}
+          >
+            Reverse
+          </button>
+          {(isResetVisible) && (
```
Comment (ArturErmolenko) [inline src/App.tsx:131]:
```suggestion
          {isResetVisible && (
```

**Review by ArturErmolenko — APPROVED** [review]

**Pull request closed without merging**